File: Easy/Soal 2/soal2.py
class Solution(object):
    def isPalindrome(self, x):
        y = str(x)
        z = ['a' for i in range(len(y))]
        p = 0
        u = 0
        for i in range(len(y) - 1,-1, -1):
            z[p] = y[i]
            p += 1
        for i in range(len(y)):
            if z[i] == y[i]:
                u += 1
        if u == len(y):
            return True
        else:
            return False


    

File: Easy/Soal 2/soal2.cpp
class Solution {
    public:
        bool isPalindrome(int x) {
            if (x < 0){
                return false;
            }
            long long test = x;
            long long rev = 0;

            while(x > 0){
                rev = (rev * 10) + (x % 10);
                x = x / 10;
            }
            if (rev == test){
                return true;
            }else{
                return false;
            }
        }
};